main.rb

```diff
@@ -0,0 +1,34 @@
+def bubble_sort(numbers)
+  switch = true
+  n = numbers.length - 1
+  while switch == true
```

Comment:
well, the switch contains a boolean value. So you don't need to compare it with ==
```ruby
while switch  #can do the trick.
```